apps/condo/domains/common/components/BaseSearchInput/index.tsx

```diff
@@ -138,6 +138,14 @@ export const BaseSearchInput = <S extends string>(props: ISearchInput<S>) => {
         onChange(value, options)
     }, [onChange])
 
+    useEffect(()=>{
```

Comment:
Write a comment.

Reply:
Wrote in new pr. https://github.com/open-condo-software/condo/pull/1514